src/index.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable linebreak-style */
```

Comment:
- [ ] As required by Microverse, you are not meant to use any linters diable rule in your codebase. The linters are there to make sure your code meets the best standard practices. I'll suggest you remove this linter disable rule and fix the issue. :+1: :+1: :+1: